### DISCUSSION TITLE
合集类页面右侧的播放列表和弹幕列表展开收起不关联

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1378] [功能建议 / Ideas]
发现合集类页面有个挺迷的特性，右侧的播放列表和弹幕列表只能展开一个，另一个必定会收起。
如果开了 `自动展开弹幕列表` 功能，就会导致一切 P 弹幕列表就展开，播放列表就收起。

所以希望合集类页面右侧的播放列表和弹幕列表展开收起是不关联的，这样就能更自由，如能同时展开或收起播放列表和弹幕列表。

**Comment by Geeyun-JY3** [comment]
v12.0 好像没用啊

**Reply by the1812** [reply]
没做啊  这个行为我干扰不了
新增的那个是合集类不自动展开弹幕列表

**Reply by Geeyun-JY3** [reply]
这样啊……就是说这个 feature 暂时实现不了？

**Reply by the1812** [reply]
是啊, 反正现在功能也能解决播放列表老是自动收起的问题

**Reply by Geeyun-JY3** [reply]
好的吧

**Comment by Geeyun-JY3** [comment]
现在新版播放页面的播放列表应该都没这特性了（也不知道是不是在加入合集的时候就已经改了）。

话说现在 discussion 能 close 了：https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/
官方说是和 close issue 和 PR 差不多，昨天试着 close 了两个 discussion，感觉区别不大。
需要直接 close 掉吗？

说起来要是用起这个 close 的 reason，至少 `feat: completed` 就可以被 `Close as resolved` 代替了吧。

**Reply by the1812** [reply]
我觉得可以用起来，比起切 label，用 close 的话订阅者能够收到通知

**Reply by Geeyun-JY3** [reply]
能收到通知那还好，只是 discussion 这种结构不像 issue 和 PR 那样划条分界线，也不知道 GitHub 后续怎么搞了😂。

那我去 close 掉那些我开的已完成的 discussion 了。
